src/main/java/com/eternalcode/lobbyheads/head/HeadInfo.java

```diff
@@ -0,0 +1,55 @@
+package com.eternalcode.lobbyheads.head;
+
+import org.bukkit.Location;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+public class HeadInfo {
+
+    private final Location location;
```

Comment:
Location -> Position

Reply:
A to nie powinno być tylko dla configów? Bo serializer deserializuje z position na location.

Reply:
Ale still masz tu memory leak'a